docs/internals/contributing/writing-code/submitting-patches.txt

```diff
@@ -290,7 +290,8 @@ All code changes
 
 * Does the :doc:`coding style
   </internals/contributing/writing-code/coding-style>` conform to our
-  guidelines? Are there any ``flake8`` errors?
+  guidelines? Are there any ``flake8`` errors? It is recommended to
```

Comment:
suggested wording: You can install the pre-commit hooks to automatically catch ...